mlir/utils/jenkins/Jenkinsfile

```diff
@@ -1057,5 +1058,68 @@ pipeline {
                 }
             }
         }
+
+        stage ("Code coverage") {
+            when {
+                beforeAgent true;
+                equals expected: true, actual: params.runCodeCoverage;
+            }
+            matrix {
+                axes {
+                    axis {
+                        name 'CHIP'
```

Comment:
Might as well account for target differences.  Can be expanded or removed.

Reply:
Would it make sense to move this into the existing PR build stages, or do those not fit?

Reply:
And I see in the document that Manupa just posted that we want to enable it for pull requests.  I think it still makes sense to have as a separate stage, since the compilation options are different than usual.